test/test_train_mp_imagenet.py

```diff
@@ -81,7 +96,33 @@
     momentum=0.9,
     lr=0.1,
     target_accuracy=0.0,
+    persistent_workers=False,
+    prefetch_factor=16,
+    loader_prefetch_size=8,
+    device_prefetch_size=4,
+    num_workers=8,
+    host_to_device_transfer_threads=1,
 )
+
+#  Best config to achieve peak performance on TPU v4
+#    1. It is recommended to use this config in conjuntion with XLA_USE_BF16=1 Flag.
+#    2. Hyperparameters can be tuned to further improve the accuracy.
+
+OPTIMIZED_KWARGS_v4 = dict(
```

Comment:
how do user run this test with v4 kwargs? Can it be done by a flag?

Reply:
Added a flag and sample command to show how to run.